app/controllers/supervisor_runs_controller.rb

```diff
@@ -1,19 +1,33 @@
 require 'json'
 
-class SupervisorScriptsController < ApplicationController
+class SupervisorRunsController < ApplicationController
   skip_before_action :verify_authenticity_token
 
+  def index
+    # TODO
+  end
+
+=begin
+  @api {get} /supervisor_runs New SupervisorRun view
+  @apiName supervisor_runs#new
+  @apiGroup SupervisorRuns
+  @apiDescription Returns partial form to configure SupervisorRun
+
+  @apiParam {String} supervisor_id ID of Supervisor to show view
+
+=end
   def new
+    redirect_to :controller=>'supervisors', :action => 'start_panel', :id => params[:supervisor_id]
```

Comment:
Należy zaznaczyć w dokumentacji, że ta metoda robi redirect (302). Możesz poszukać czy w apidocjs nie da się napisać o tym, że to jest redirect.